### ISSUE TITLE
Implement null constant

### ISSUE THREAD

**Issue by wolfgang42** [wolfgang42/rockstar-js#29]
Make sure constants all match spec.

> - Constants: `mysterious`, `null`, `nothing`, `nowhere`, `nobody`, `empty`, `gone`, `true`, `right`, `yes`, `ok`, `false`, `wrong`, `no`, `lies`, `maybe`, `definitely maybe`

**Comment by wolfgang42** [comment]
`maybe`/`definitely maybe` are reserved but have no implementation. All others except `null` were already implemented.

**Comment by ldevernay** [comment]
How do you want to treat reserved words?
Do we wait for their implementation before taking them into account?

**Comment by wolfgang42** [comment]
Yes, there's nothing to be done for them until a behaviour is specified. I've updated the title accordingly.
